src/main/java/org/wso2/carbon/module/swiftiso20022/validation/JsonToMT103PayloadValidator.java

```diff
@@ -0,0 +1,415 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ * <p>
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.module.swiftiso20022.validation;
+
+import org.wso2.carbon.module.swiftiso20022.constants.ConnectorConstants;
+import org.wso2.carbon.module.swiftiso20022.constants.MT103Constants;
+import org.wso2.carbon.module.swiftiso20022.validation.common.ValidationEngine;
+import org.wso2.carbon.module.swiftiso20022.validation.common.ValidatorContext;
+import org.wso2.carbon.module.swiftiso20022.validation.rules.custom.MT103ArrayFieldValidationRule;
+import org.wso2.carbon.module.swiftiso20022.validation.rules.custom.MT103EntityFieldValidationRule;
+
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Class to validate the JSON payload for MT103 transformation.
+ */
+public class JsonToMT103PayloadValidator {
+
+    private static final List<ValidatorContext> block01MandatoryFieldValidationParamList = List.of(
```

Comment:
Shall we add comments for methods in this class

Reply:
Added in commit 4a2dcd837e62caad7075e4185b59627cb83ad08d